### PR TITLE
Move `LocalQueries` commands to be member functions

### PR DESCRIPTION
**Pull request by dbartol**
This PR implements a [review suggestion](https://github.com/github/vscode-codeql/pull/2244#discussion_r1150312991) from #2244. I've made it a separate, dependent PR to avoid making the original PR's diffs even worse.

### CODE REVIEW

**Comment by aeisenberg** [comment]
Just wanting to be clear. `[dbartol/debug-adapter](https://github.com/github/vscode-codeql/tree/dbartol/debug-adapter)` has no PR associated with it. Is this PR targeting the right branch?

File: extensions/ql-vscode/test/vscode-tests/no-workspace/contextual/astBuilder.test.ts [hunk 1]
```diff
@@ -52,11 +53,12 @@ describe("AstBuilder", () => {
     const astBuilder = createAstBuilder();
     const roots = await astBuilder.getRoots();
 
+    const bqrsPath = path.normalize("/a/b/c/results.bqrs");
```
Comment (aeisenberg) [inline extensions/ql-vscode/test/vscode-tests/no-workspace/contextual/astBuilder.test.ts:56]: Any understanding of why this test had to be changed?

**Comment by dbartol** [comment]
> Just wanting to be clear. `[dbartol/debug-adapter](https://github.com/github/vscode-codeql/tree/dbartol/debug-adapter)` has no PR associated with it. Is this PR targeting the right branch?

Oh, yikes! That's the wrong target branch. I'll fix this in a few minutes.

File: extensions/ql-vscode/test/vscode-tests/no-workspace/contextual/astBuilder.test.ts (hunk 1 above)
Comment (dbartol) [reply extensions/ql-vscode/test/vscode-tests/no-workspace/contextual/astBuilder.test.ts]: I think that change went into the base PR, but the reason for the change was that previously, the test code passed in `"/a/b/c"` as the path to the BQRS file itself, which got passed through the implementation verbatim and came out the other end with forward slashes intact. With my changes, `"/a/b/c"` is treated as the query output directory, which goes through some `path` calls (`join`, etc.) before the BQRS path comes out the other end with platform-specific separators.

**Pull request merged**